frontend/src/pages/ActivityReport/Pages/components/NextStepsRepeater.js

```diff
@@ -76,68 +93,124 @@ export default function NextStepsRepeater({
     }
   };
 
+  const stepType = name === 'specialistNextSteps' ? 'specialist' : 'recipient';
+
   return (
     <>
       <div className="ttahub-next-steps-repeater">
-        { fields.map((item, index) => (
-          <FormGroup
-            key={`next-step-form-group-${index + 1}`}
-            className="margin-top-1"
-            error={blurValidations[index]}
-          >
-            {
-                blurValidations[index] || (errors[name] && errors[name][index])
-                  ? <ErrorMessage>Enter a next step</ErrorMessage>
-                  : null
-                }
-            <div
-              key={`next-step-flex-${index + 1}`}
-              className={`display-flex ${blurValidations[index] || (errors[name] && errors[name][index]) ? 'blank-next-step' : ''}`}
+        {fields.map((item, index) => (
+          <div key={`${stepType}-parent-div-${index + 1}`}>
+            <FormGroup
+              key={`${stepType}-next-step-form-group-step-${index + 1}`}
+              className="margin-top-2 margin-bottom-2"
+              error={blurStepValidations[index] || (errors[name] && errors[name][index]
+                && errors[name][index].note)}
             >
-              <Label
-                htmlFor={`${name === 'specialistNextSteps'
-                  ? 'specialist' : 'recipient'}-next-step-${index + 1}`}
-                className="sr-only"
+              <legend>
```

Comment:
- You can remove the legends here, they only belong in ```<fieldset>```.

- I think the labels should just reflect the actual text you see. (When do you anticipate completing this step?) We should consult the design team and ask if we can change that to When do you anticipate completing step 1?, etc.

- it looks like the colon is in the wrong place - it should be "Step 1**:** *" not "Step**:** 1 *"